src/middlewared/middlewared/plugins/failover_/disabled_reasons.py

```diff
@@ -26,6 +26,7 @@ class DisabledReasonsEnum(str, Enum):
     LOC_FAILOVER_ONGOING = 'This node is currently processing a failover event.'
     NO_HEARTBEAT_IFACE = 'Local heartbeat interface does not exist.'
     NO_CARRIER_ON_HEARTBEAT = 'Local heartbeat interface is down.'
+    REBOOT_REQUIRED_FOR_FIPS = 'Reboot required for FIPS configuration change to take effect'
```

Comment:
This won't work on HA. The customer/support team is going to have to know which node needs to be rebooted.

```
LOC_FIPS_REBOOT_REQ = 'This node needs to be rebooted to apply FIPS configuration'
REM_FIPS_REBOOT_REQ = 'Other node needs to be rebooted to apply FIPS configuration'
```